diff --git a/tree-sitter-langs.el b/tree-sitter-langs.el
index 2d4468c..7baf2c2 100644
--- a/tree-sitter-langs.el
+++ b/tree-sitter-langs.el
@@ -53,11 +53,6 @@
   "Grammar bundle for `tree-sitter'."
   :group 'tree-sitter)

-(defvar tree-sitter-langs--testing)
-(eval-and-compile
-  (unless (bound-and-true-p tree-sitter-langs--testing)
-    (tree-sitter-langs-install-grammars :skip-if-installed)))
-
 (defun tree-sitter-langs-ensure (lang-symbol)
   "Return the language object identified by LANG-SYMBOL.
 If it cannot be loaded, this function tries to compile the grammar.
