diff --git a/lisp/tree-sitter.el b/lisp/tree-sitter.el
index a854bda..4787b99 100644
--- a/lisp/tree-sitter.el
+++ b/lisp/tree-sitter.el
@@ -6,7 +6,7 @@
 ;; Keywords: languages tools parsers tree-sitter
 ;; Homepage: https://github.com/ubolonton/emacs-tree-sitter
 ;; Version: 0.15.1
-;; Package-Requires: ((emacs "25.1") (tsc "0.15.1"))
+;; Package-Requires: ((emacs "25.1"))
 ;; SPDX-License-Identifier: MIT

 ;;; Commentary:
